### ISSUE TITLE
Nil error on user profile (Logs P2)

### ISSUE THREAD

**Issue by benwbrum** [benwbrum/fromthepage#2064]
From logs
```

production.log-I, [2020-09-18T08:07:00.831802 #30111]  INFO -- : Started GET "/ftpalmmsa" for 157.55.39.88 at 2020-09-18 08:07:00 +0000
production.log-I, [2020-09-18T08:07:00.834641 #30111]  INFO -- : Processing by UserController#profile as */*
production.log-I, [2020-09-18T08:07:00.835012 #30111]  INFO -- :   Parameters: {"user_id"=>"ftpalmmsa"}
production.log-I, [2020-09-18T08:07:00.836907 #30111]  INFO -- : [ahoy] Visit excluded
production.log-I, [2020-09-18T08:07:00.858264 #30111]  INFO -- :   Rendering user/profile.html.slim within layouts/application
production.log-I, [2020-09-18T08:07:00.920189 #30111]  INFO -- :   Rendered user/profile.html.slim within layouts/application (Duration: 61.1ms | Allocations: 5759)
production.log:I, [2020-09-18T08:07:00.922193 #30111]  INFO -- : Completed 500  in 87ms (ActiveRecord: 46.8ms | Allocations: 12770)
production.log-F, [2020-09-18T08:07:00.936171 #30111] FATAL -- :   
production.log-ActionView::Template::Error (undefined method `work' for nil:NilClass):
production.log-    57:         tr
production.log-    58:           td.nowrap =version.created_on.localtime.strftime("%b %d, %Y, %l:%M %p")
production.log-    59:           td.overflow
production.log-    60:             -if version.page.work.present?
production.log-    61:               =link_to version.page.work.title, collection_read_work_path(version.page.collection.owner, version.page.collection, version.page.work)
production.log-    62:             -else
production.log-    63:               span.a50 =t('.no_work')
production.log-  
production.log-app/views/user/profile.html.slim:60
production.log-app/views/user/profile.html.slim:56
production.log-app/controllers/application_controller.rb:29:in `switch_locale'
```

**Comment by benwbrum** [comment]
Another example:
```

production.log-I, [2020-09-18T08:20:06.590563 #31205]  INFO -- : Processing by UserController#profile as */*
production.log-I, [2020-09-18T08:20:06.591027 #31205]  INFO -- :   Parameters: {"user_id"=>"215431"}
production.log-I, [2020-09-18T08:20:06.592953 #31205]  INFO -- : [ahoy] Visit excluded
production.log-I, [2020-09-18T08:20:06.625334 #31205]  INFO -- :   Rendering user/profile.html.slim within layouts/application
production.log-I, [2020-09-18T08:20:06.652701 #31226]  INFO -- :   Rendered shared/_collection_footer.html.slim (Duration: 0.3ms | Allocations: 56)
production.log-I, [2020-09-18T08:20:06.653261 #31226]  INFO -- :   Rendered display/list_pages.html.slim within layouts/application (Duration: 195.8ms | Allocations: 66790)
production.log-I, [2020-09-18T08:20:06.655320 #31226]  INFO -- :   Rendered layouts/_mixpanel.erb (Duration: 0.2ms | Allocations: 54)
production.log-I, [2020-09-18T08:20:06.656155 #31226]  INFO -- :   Rendered layouts/_fb_pixel.erb (Duration: 0.1ms | Allocations: 54)
production.log-I, [2020-09-18T08:20:06.662051 #31226]  INFO -- :   Rendered shared/_breadcrumbs.html.slim (Duration: 0.5ms | Allocations: 102)
production.log-I, [2020-09-18T08:20:06.664007 #31226]  INFO -- :   Rendered layouts/_ga.erb (Duration: 0.2ms | Allocations: 54)
production.log-I, [2020-09-18T08:20:06.664773 #31226]  INFO -- : [ahoy] Event excluded
production.log-I, [2020-09-18T08:20:06.665935 #31226]  INFO -- : Completed 200 OK in 230ms (Views: 187.8ms | ActiveRecord: 24.5ms | Allocations: 76820)
production.log-I, [2020-09-18T08:20:06.666299 #31226]  INFO -- : Oink Action: display#list_pages
production.log-I, [2020-09-18T08:20:06.666547 #31226]  INFO -- : Memory usage: 1764408 | PID: 31226
production.log-I, [2020-09-18T08:20:06.666800 #31226]  INFO -- : Instantiation Breakdown: Total: 51 | Work: 24 | Page: 23 | Collection: 2 | PageBlock: 1 | User: 1
production.log-I, [2020-09-18T08:20:06.666972 #31226]  INFO -- : Oink Log Entry Complete
production.log-I, [2020-09-18T08:20:06.675089 #31205]  INFO -- :   Rendered user/profile.html.slim within layouts/application (Duration: 48.9ms | Allocations: 4874)
production.log:I, [2020-09-18T08:20:06.677405 #31205]  INFO -- : Completed 500  in 86ms (ActiveRecord: 40.4ms | Allocations: 14934)
production.log-F, [2020-09-18T08:20:06.691038 #31205] FATAL -- :   
production.log-ActionView::Template::Error (undefined method `work' for nil:NilClass):
production.log-    57:         tr
production.log-    58:           td.nowrap =version.created_on.localtime.strftime("%b %d, %Y, %l:%M %p")
production.log-    59:           td.overflow
production.log-    60:             -if version.page.work.present?
production.log-    61:               =link_to version.page.work.title, collection_read_work_path(version.page.collection.owner, version.page.collection, version.page.work)
production.log-    62:             -else
production.log-    63:               span.a50 =t('.no_work')
production.log-  
production.log-app/views/user/profile.html.slim:60
production.log-app/views/user/profile.html.slim:56
production.log-app/controllers/application_controller.rb:29:in `switch_locale'
```

**Comment by benwbrum** [comment]
Resolved in fix for #2392
